tck/features/clauses/return-orderby/Return-Orderby1.feature

```diff
@@ -1,367 +1,236 @@
-    And no side effects
-
-  Scenario: ORDER BY DESC should order distinct types in the expected order
-    And having executed:
-      """
-      CREATE (:N)-[:REL]->()
-      """
-    When executing query:
-      """
-      MATCH p = (n:N)-[r:REL]->()
-      UNWIND [n, r, p, 1.5, ['list'], 'text', null, false, 0.0 / 0.0, {a: 'map'}] AS types
-      RETURN types
-      ORDER BY types DESC
-      """
-    Then the result should be, in order:
-      | types             |
-      | null              |
-      | NaN               |
-      | 1.5               |
-      | false             |
-      | 'text'            |
-      | <(:N)-[:REL]->()> |
-      | ['list']          |
-      | [:REL]            |
-      | (:N)              |
-      | {a: 'map'}        |
-    And no side effects
-
-  Scenario: Handle ORDER BY with LIMIT 1
-    And having executed:
-      """
-      CREATE (s:Person {name: 'Steven'}),
-        (c:Person {name: 'Craig'})
-      """
-    When executing query:
-      """
-      MATCH (p:Person)
-      RETURN p.name AS name
-      ORDER BY p.name
-      LIMIT 1
-      """
-    Then the result should be, in order:
-      | name    |
-      | 'Craig' |
-    And no side effects
-
-  Scenario: ORDER BY with LIMIT 0 should not generate errors
-    When executing query:
-      """
-      MATCH (p:Person)
-      RETURN p.name AS name
-      ORDER BY p.name
-      LIMIT 0
-      """
-    Then the result should be, in order:
-      | name |
-    And no side effects
+#
+# Copyright (c) 2015-2020 "Neo Technology,"
+# Network Engine for Objects in Lund AB [http://neotechnology.com]
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Attribution Notice under the terms of the Apache License 2.0
+#
+# This work was created by the collective efforts of the openCypher community.
+# Without limiting the terms of Section 6, any Derivative Work that is not
+# approved by the public consensus process of the openCypher Implementers Group
+# should not be described as “Cypher” (and Cypher® is a registered trademark of
+# Neo4j Inc.) or as "openCypher". Extensions by implementers or prototypes or
+# proposals for change that have been documented or implemented should only be
+# described as "implementation extensions to Cypher" or as "proposed changes to
+# Cypher that are not yet approved by the openCypher community".
+#
+
+#encoding: utf-8
+
+Feature: Return-Order By 1 - Order by a single variable (correct order of values according to their type)

```

Comment:
```suggestion
Feature: ReturnOrderBy1 - Order by a single variable (correct order of values according to their type)
```
For consistency. In `MatchWhereX` and  `WithWhereX` we also have not used hyphens or spaces on the feature name.